pyproject.toml

```diff
@@ -1,7 +1,63 @@
+[project]
+name = "comtypes"
+description = "Pure Python COM package"
+readme = "README.md"
+requires-python = ">=3.9"
+license = "MIT"
+license-files = ["LICENSE.txt"]
+authors = [
+    { name = "Thomas Heller", email = "theller@python.net" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
```

Comment:
We might migrate the `License :: OSI Approved :: MIT License` setting to `pyproject.toml`'s `classifiers`, as it was previously configured in `setup.py` or `setup.cfg`.

https://github.com/enthought/comtypes/blob/938fbd50d2077c6bd524790a9ed798c3990cdfc0/setup.py#L18-L27

https://github.com/enthought/comtypes/blob/fb8fd5823d83f866df43ce6f15d1b8dfd7f51f8c/setup.cfg#L11-L18

Reply:
If i do that and run ``python -m build``, I get this error:
```
setuptools.errors.InvalidConfigError: License classifiers have been superseded by license expressions (see https://peps.python.org/pep-0639/). Please remove:

License :: OSI Approved :: MIT License

ERROR Backend subprocess exited when trying to invoke get_requires_for_build_sdist
```

This is why I didn't included it.